File: src/main/java/fi/finlit/edith/domain/NoteStatus.java
/*
 * Copyright (c) 2009 Mysema Ltd.
 * All rights reserved.
 *
 */
package fi.finlit.edith.domain;

import com.mysema.rdfbean.annotations.ClassMapping;

@ClassMapping
public enum NoteStatus {
    /**
     *
     */
    INITIAL,

    /**
     *
     */
    DRAFT,

    /**
     *
     */
    FINISHED;
}

File: src/main/scala/fi/finlit/edith/sql/domain/NoteStatus.scala
package fi.finlit.edith.sql.domain

//remove if not needed
import scala.collection.JavaConversions._

object NoteStatus {

  class NoteStatus {
  }

  val INITIAL = new NoteStatus()

  val DRAFT = new NoteStatus()

  val FINISHED = new NoteStatus()
}
